app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -726,6 +727,37 @@ class BrowserTabViewModel @Inject constructor(
         }
     }
 
+    fun userLongPressedAutocomplete(suggestion: AutoCompleteSuggestion) {
+        when (suggestion) {
+            is AutoCompleteHistorySuggestion -> showRemoveSearchSuggestionDialog(suggestion)
+            is AutoCompleteHistorySearchSuggestion -> showRemoveSearchSuggestionDialog(suggestion)
```

Comment:
nit: Since the action is the same in both cases, you can do

` is AutoCompleteHistorySuggestion, is AutoCompleteHistorySearchSuggestion -> showRemoveSearchSuggestionDialog(suggestion)`

Reply:
I didn’t know you could do that, TIL 👍